docs/development/affiliated-packages.rst

```diff
@@ -399,9 +399,9 @@ files manually`_ section since this explains what many of the files do.
    replace ``packagename`` with the name of your package.
 
 #. If you want the documentation for your project to be hosted by
-   `ReadTheDocs <https://readthedocs.org>`_, then you need to setup an
+   `Read The Docs <https://readthedocs.org>`_, then you need to setup an
    account there. The following entries in "Advanced Settings" for your
-   package on `ReadTheDocs <https://readthedocs.org>`_ should work:
+   package on `Read The Docs <https://readthedocs.org>`_ should work:
```

Comment:
"The" should be "the" (not capitalized). I am following their official title on https://readthedocs.org/ .